Cargo.toml

```diff
@@ -12,10 +12,11 @@ test = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "1.0.136", features = ["derive", "rc"] }
```

Comment:
Just wanted to make sure... does this look ok? I put this in here and it might have flown under Jonathan's radar.

Reply:
I think the only thing that I will change here is stop fixing such a specific version, 1.0 should be good enough,